packages/aa/package.json

```diff
@@ -11,7 +11,7 @@
   },
   "scripts": {
     "test": "npm run test:run",
-    "test:run": "ava --timeout=30s test/index.js",
+    "test:run": "ava",
```

Comment:
(can be ignored) IMHO the arguments are more convenient as one can copy the call and modify/extend without thinking of the configuration elsewhere, but I'm ok with this change as it provides benefits elsewhere.